sencha-workspace/packages/slate-cbl/src/view/demonstrations/SkillList.js

```diff
@@ -79,28 +92,27 @@ Ext.define('Slate.cbl.view.demonstrations.SkillList', {
                         '</div>',
                     '</td>',
                     '<tpl if="Override">',
-                        '<td colspan="3" class="skill-list-demo-data skill-list-override">Override</td>',
+                        '<td colspan="2" class="skill-list-demo-data skill-list-override">[Overridden]</td>',
                     '<tpl else>',
-                        '<td class="skill-list-demo-data skill-list-demo-type">{Demonstration.ExperienceType:htmlEncode}</td>',
-                        '<td class="skill-list-demo-data skill-list-demo-context">{Demonstration.Context:htmlEncode}</td>',
-                        '<td class="skill-list-demo-data skill-list-demo-task">{Demonstration.PerformanceType:htmlEncode}</td>',
+                        '<td class="skill-list-demo-data skill-list-demo-experience">',
+                            '<div class="skill-list-title">{Demonstration.Context:htmlEncode}</div>',
+                            '<small class="skill-list-subtitle">{Demonstration.ExperienceType:htmlEncode}</small>',
+                        '</td>',
+                        // '<td class="skill-list-demo-data skill-list-demo-type" data-qtip="{Demonstration.ExperienceType:htmlEncode}">{Demonstration.ExperienceType:htmlEncode}</td>',
+                        // '<td class="skill-list-demo-data skill-list-demo-context" data-qtip="{Demonstration.Context:htmlEncode}">{Demonstration.Context:htmlEncode}</td>',
+                        '<td class="skill-list-demo-data skill-list-demo-task">',
+                            '<a class="skill-list-linkable" href="#">',
+                                '<i class="linkable-icon fa fa-external-link-square"></i>',
+                                '<div class="linkable-content">',
+                                    '{Demonstration.PerformanceType:htmlEncode}',
```

Comment:
this should be `Demonstration.StudentTask.Task.Title` and this whole link should be wrapped in `<tpl if="Demonstration.StudentTask">`

Reply:
These changes have been made